src/SmartComponents/HybridInventoryTabs/HybridInventoryTabs.js

```diff
@@ -13,6 +14,8 @@ const ConventionalSystems = lazy(() =>
 
 const HybridInventory = (props) => {
   const isEdgeParityEnabled = useFeatureFlag('advisor.edge_parity');
+  const { hasEdgeDevices, hasConventionalSystems } =
+    useContext(AccountStatContext);
```

Comment:
Why the need for context? You're not changing the values and only using the defaults

Reply:
Thank you @AsToNlele for the review! Yeah, my bad I forgot to add the context provider that sets context values to be shared across components. Now it is added :)

Reply:
And my commit ^ dismissed your approval. Please have a quick look one more time